runtime/src/accounts_db.rs

```diff
@@ -3040,11 +3044,11 @@ impl AccountsDB {
         let old_slots = self.accounts_cache.find_older_frozen_slots(MAX_CACHE_SLOTS);
         let excess_slot_count = old_slots.len();
         let mut unflushable_unrooted_slot_count = 0;
-        let max_flushed_root = self.accounts_cache.fetch_max_flush_root();
+        let max_previously_flushed_root = self.accounts_cache.fetch_max_flush_root();
```

Comment:
nits: I think `old_max_flush_root` or `prev_max_flush_root` follows the convention of coding (`old_` or `prev_` prefix) to keep old value in advance of possible mutation of the value.

Mixing `_previously_` in middle sounds something more convoluted is about to happen.

Reply:
With your suggestion to switch `max_flush_root -> requested_flush_root`, this can go back to being `max_flushed_root` without ambiguity